ipi/inputs/simulation.py

```diff
@@ -273,9 +291,10 @@ def fetch(self):
             if k == "system":
                 syslist.append(v.fetch())
             elif k == "system_template":
-                syslist += (
```

Comment:
Why this change? I think the multi-line comment was more readable

Reply:
Not sure what you mean. I noticed that Black exploded a simple expression over many lines because of long comment on the same line, so I moved the comment above the expression and split manually. Mainly to prevent exploding the expression.